lib/internal/bootstrap_node.js

```diff
@@ -504,7 +513,7 @@
   };
 
   NativeModule.wrapper = [
-    '(function (exports, require, module, __filename, __dirname) { ',
+    '(function (exports, require, module, __filename, process) { ',
```

Comment:
Am I reading this correctly, internal modules no longer have `__dirname`? I guess it would be too big a change to have `process` be injected as a module argument for all modules, internal and external, rather than be in `globals`? Ben, was there discussion about whether node's "globals" should be true-global or module locals? module/exports/__filename/__dirname have meaning only in local context, but why did `require()` become a local?